ruma-events/tests/any_enums.rs

```diff
@@ -0,0 +1,278 @@
+use std::convert::TryFrom;
```

Comment:
I don't like the name of this test file. If it is testing specifically `ruma_events::enums`, maybe just name it `enums` too?